web/src/pages/CreateTest/CreateTestContent.tsx

```diff
@@ -6,12 +7,24 @@ import * as S from './CreateTest.styled';
 
 const CreateTestContent: React.FC = () => {
   const navigate = useNavigate();
-  const {stepList, activeStep, pluginName, onGoTo, isLoading} = useCreateTest();
+  const {stepList, activeStep, pluginName, onGoTo, onReset, isLoading} = useCreateTest();
+
+  useEffect(() => {
+    return () => {
+      onReset();
```

Comment:
Resetting the page before unmount

Reply:
cant we have this reset inside the onSubmit event to avoid reseting state within useEffect?

Reply:
The idea of the use effect is to reset no matter the step, if the user was on the second step it wasn't resetting correctly. If add it to the `onSubmit` we won't be covering the user leaving mid way